src/msg/async/ProtocolV1.cc

```diff
@@ -240,6 +240,10 @@ void ProtocolV1::send_message(Message *m) {
     out_q[m->get_priority()].emplace_back(std::move(bl), m);
     ldout(cct, 15) << __func__ << " inline write is denied, reschedule m=" << m
                    << dendl;
+    // Fixme: for write_message() > 0 and write_in_progress is true, we don't add external_event
```

Comment:
It would probably make more sense to add a comment at the ```break``` statement for the ```write_message() > 0``` case. Then you could just say something succinct like "outbound message in-progress, thread will be re-awoken when the outbound thread is writeable again"